### PR TITLE
CASMNET-2134 | Improve network install docs

### PR DESCRIPTION
**Pull request by kosonenj**
# Description

Improvements to network management to make installation and finding resources easier. 

https://jira-pro.it.hpe.com:8443/browse/CASMNET-2134

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/network/management_network/README.md [hunk 1]
```diff
@@ -24,21 +24,34 @@ exactly match any particular environment. The switch and
 accessory drawings in this document are for illustration only, and may not exactly match installed
 products.
 
+## Install scenarios
+
+| Situation                                                                       | Link                                              |
+|---------------------------------------------------------------------------------|---------------------------------------------------|
+| Upgrade CANU to the latest version                                              | [Install/Upgrade CANU](canu_install_update.md)    |
+| Upgrade to CANU-generated configurations from non-CANU-generated configurations | [Upgrade](upgrade.md)                             |
+| Current switch configurations are CANU-generated and need to go to 1.3          | [`1.3 (Preconfig)` to 1.3](1.2_to_1.3_upgrade.md) |
```
Comment (lukebates123) [inline operations/network/management_network/README.md:33]: We should probably update or remove the CSM version numbers throughout the net docs.

**Review by lukebates123 — CHANGES_REQUESTED** [review]

**Comment by kosonenj** [comment]
Lintcheck fails because of: 

2 links checked.
  [✖] ../../troubleshooting/known_issues/qlogic_driver_crash.md → Status: 400
  
 PR https://github.com/Cray-HPE/docs-csm/pull/3822 Needs to be merged first

File: operations/network/management_network/firmware/update_management_network_firmware.md [hunk 2]
```diff
@@ -13,10 +20,10 @@ All firmware can be found in the HFP package provided with the Shasta release.
...
 | ----- | -----: |
-| Aruba 8320 Switch Series | 10.09.0010  |
-| Aruba 8325 Switch Series | 10.09.0010  |
-| Aruba 8360 Switch Series | 10.09.0010  |
-| Aruba 6300 Switch Series | 10.09.0010  |
+| Aruba 8320 Switch Series | 10.11.1010  |
+| Aruba 8325 Switch Series | 10.11.1010  |
+| Aruba 8360 Switch Series | 10.11.1010  |
+| Aruba 6300 Switch Series | 10.11.1010  |
 | Mellanox SN2100 Switch Series | 3.9.3210|
```
Comment (rustydb) [inline operations/network/management_network/firmware/update_management_network_firmware.md:27]: Same comment https://github.com/Cray-HPE/docs-csm/pull/3814#discussion_r1222053008.

**Review by rustydb — CHANGES_REQUESTED** [review]

File: install/csm-install/README.md [hunk 3]
```diff
@@ -88,6 +89,65 @@ See [Create system configuration](../pre-installation.md#3-create-system-configu
 
 ### 5. Configure management network switches
 
+#### Management network
```
Comment (denniswalker) [inline install/csm-install/README.md:92]: Can we move this to its own document and then link here and also from install/common-pre-install/README.md#configure-the-management-network-with-canu

Separately, is this the same set of concerns for upgrades?

**Review by nrockershousen — APPROVED** [review]

**Comment by rustydb** [comment]
Rebased on main after merging #3822 and #3814

**Comment by kosonenj** [comment]
closing because of merge conficts from pet changes and want to make cleaner PR

**Pull request closed without merging**